Add core dump docs for PHP in docker (#19820)
DataDog/documentation: content/en/tracing/trace_collection/dd_libraries/php.md

1. Ensure you have a suitable `ulimit` set in your system. You can set it to unlimited: `ulimit -c unlimited`.
1. If your application runs in a Docker container, changes to `/proc/sys/*` have to be done to the host machine. Contact your system administrator to know the options available to you. If you are able to, try recreating the issue in your testing or staging environments.

> ### Obtaining a core dump from within a docker container

Reviewer: ```suggestion
### Obtaining a core dump from within a Docker container
```

> Obtaining a core dump in a docker container adds another layer of challenges to overcome. Here are a few tips to help you obtain a core dump:

Reviewer: ```suggestion
Use the information below to assist with obtaining a core dump in a Docker container:
```

> 1. The docker container needs to run as a privileged container and the ulimit for core files needs to be maxed out.

Reviewer: ```suggestion
1. The Docker container needs to run as a privileged container, and the `ulimit` value for core files needs to be set to its maximum as shown in the examples below.
```

>    - In case you use `docker run` add the `--privileged` and the `--ulimit core=99999999999` argument
>    - In case you use `docker compose` add the following to the container in the `docker-compose.yml`:

Reviewer: ```suggestion
   - If you use the `docker run` command, add the `--privileged` and the `--ulimit core=99999999999` arguments
   - If you use `docker compose`, add the following to the `docker-compose.yml` file:
```
